### DISCUSSION TITLE
Explicitly fallthrough a switch condition

### DISCUSSION THREAD

**Discussion by asumbek** [PlutoLang/Pluto#1031] [Ideas]
Hello, I would like to propose a feature change on `switch` statement.

Instead of explicitly tells the switch condition to break. Make them to break by default and use a keyword to explicitly "falls through" from a switch condition to another.
```lua
local x = 3
switch x do
  switch 1: fallthrough
  switch 2: fallthrough
  switch 3: fallthrough
  switch 4: fallthrough
  switch 5: print("In range of 1-5")
  default: print($"{x} is not in range of 1-5)
end
```
This will reduce the time to write `break` everytime you want the condition to not falls through to another condition.

**Comment by Sainan** [comment]
This was just discussed in #1028. It's not planned to make such a breaking change.
